tests/xharness/Jenkins.cs

```diff
@@ -1425,6 +1438,10 @@ public void GenerateReport ()
 							File.Delete (Harness.MarkdownSummaryPath);
 						File.Move (tmpmarkdown, Harness.MarkdownSummaryPath);
 					}
+					var dependentFileLocation = Path.GetDirectoryName (System.Reflection.Assembly.GetExecutingAssembly ().Location);
+					foreach (var file in new string [] { "xharness.js", "xharness.css" }) {
```

Comment:
Not something that should be changed in the PR, but what about, since we have a dir name, use a regex to get all *.js and *.css and do the  same operation?